### ISSUE TITLE
[TASK] Research Fields_FuzzyDate - FuzzyDateDocumentField

### ISSUE THREAD

**Issue by Libisch** [Beit-Hatfutsot/mojp-dbs-pipelines#63]
- What's the use/meaning of the other fields in _Fields_FuzzyDate_, besides "YearNumber"?
- Is there any other info or documentation regarding dates (e.g photo taken, person born, etc.), besides [this](https://www.clearmash.com/skn/c7/Support/e4013/FuzzyDateDocumentField#search=f460f6a3d2dbd6749043515f3955931f&entity_number=1)?
- API?

**Comment by Libisch** [comment]
@omrisuissa Please advise. Thanks.
@OriHoch

**Comment by omrisuissa** [comment]
Fuzzy date is a complex type that represents incomplete date. This is a concept developed for historical archives (like BH) when you have partial dates.
Before you approach using it in ClearMash you should first investigate and understand how GEDCOM represents it since ClearMash’s fuzzy date designed for GEDCOM partial dates.
I’ll be more than happy to answer specific questions regarding this topic, but please make them specific :)

**Comment by Libisch** [comment]
@nuritgazit @OriHoch 
Further explanation (after talking to @omrisuissa): 
- Fuzzy date represents the raw data as previously presented in BHP. That means invalid or unusable date formats (e.g "The '90", "1800-1900" etc.).
- In the Clearmash CSM these dates are usable and searchable, but not in the raw data we use for the website.

**Comment by nuritgazit** [comment]
@Libisch @omrisuissa Not sure I understand. So this option was implemented only for trees but not for images / other types of items?

**Comment by OriHoch** [comment]
in this case we need it for photoUnits

**Comment by Libisch** [comment]
No, it means clearmash have developed a way to make use of the dates, but
it's not available to us (as we use the raw data with our own backend).
Regardless of the item type.

**Comment by nuritgazit** [comment]
@omrisuissa is this a technical or commercial issue?
